docs/it/remote-working.md

```diff
@@ -0,0 +1,89 @@
+
+Esistono molte community online di altri nomadi digitali e siti che permettono di valutare le mete preferibili per iniziare, uno su tutti [Nomad list](https://nomadlist.com/), ma in assoluto viaggiare lavorando è una pratica sempre più diffusa che può davvero cambiare scenario alle giornate di lavoro.
+
+## Programmare da remoto funziona davvero?
+
+Se il lavoro da remoto porta molti vantaggi sulla gestione della proprio vita privata, ci sono però aspetti tecnici che infiammano le discussioni tra noi developers, con molte criticità che possono presentarsi con questa modalità di lavoro.
+
+La più evidente riguarda la formazione dei developers junior, che entrando in un'azienda completamente da remoto non hanno la possibilità di lavorare letteralmente _fianco fianco_ ai developers senior e quindi non riescono ad apprendere velocemente dall'esperienza e si ritrovano isolati in una fase molto importante della loro formazione.
+
+In realtà a questo si può in parte sopperire con uno sforzo maggiore dell'azienda e del resto del team a implementare un processo di on-boarding strutturato, sessioni dedicate esclusivamente alla formazione e permettere ai dev con meno seniority di poter facilmente avere supporto in caso di problematiche, tecniche e non.
+
+Un'altra criticità riguarda la difficoltà di coordinare il lavoro, con meeting fatti in videochiamata poco produttivi rispetto a sessioni di brain storming di persona e più in generale poca coesione all'interno del team rispetto ad un gruppo di developer che lavorano nella stessa stanza.
+
+Questa è sicuramente una possibilità concreta ed è innegabile che l'essere fisicamente con altre persone non sia la stessa cosa di una chiamata su Zoom, è anche per questo che molte aziende decidono comunque di avere una sede o degli uffici a disposizione dei dipendenti, che sono liberi di scegliere se e quando andare.
+
+Infine spesso si evidenzia un aumento del turn over all'interno delle aziende, ovvero è più facile che un developer che lavora da remoto si licenzi perché non ha creato un reale legame con i colleghi e quel senso di appartenenza che fa si che si crei un clima aziendale accogliente.
+
+Questa criticità dovrebbe essere supportata da dati più concreti, ma è dimostrabile come il lavoro in presenza non sia sinonimo di clima aziendale positivo e che molte aziende full remote riescano a creare una sinergia tra le persone grazie alla condivisione di valori e all'organizzazione di eventi in persona, per fare il famoso team building, dimostrando un reale interesse a creare un gruppo coeso e soddisfatto del proprio lavoro.
```

Comment:
Questa frase la riscriverei.. Capisco il senso, ma comincia come se fosse un proseguio della precedente mentre è letteralmente l'opposto.. Non saprei cosa consigliare, magari ti viene in mente un altro modo di esprimerlo!
Altra faccia della medaglia? D'altro canto? Non so..

Reply:
CI sta, ho provato a sottolineare l'anternanza